lib/gyro/parser/xcdatamodel/xcdatamodel.rb

```diff
@@ -37,18 +37,20 @@ class XCDataModel
 
         def initialize(xcdatamodel_dir)
           is_xcdatamodeld = xcdatamodel_dir.extname != '.xcdatamodeld'
-          Gyro::Log.fail!('Please target an xcdatamodel inside your xcdatamodeld') unless is_xcdatamodeld
+          Gyro::Log.fail!('Please target an \'.xcdatamodel\' file inside your xcdatamodeld') unless is_xcdatamodeld
```

Comment:
I'm surprised `rubocop` doesn't suggest you here to use `%q(Please target an '.xcdatamodel' file …)` syntax here (`%q(…)` allowing you to avoid having to escape the single quotes in the string)

Reply:
ok so I tried with `%q`but rubocop told that this syntax is only for string that contain both single quotes and double quotes. I seached and I found `%()` and `%Q()`.

Reply:
And rubocop tell me to use `%()` instead of `%Q()`